src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './TodoInfo.scss';
+import UserInfo from '../UserInfo';
+import { Result } from '../../react-app-env';
+
+export const TodoInfo: React.FC<Result> = ({ ...whatTodo }) => {
```

Comment:
You can take properties you need right here
```suggestion
export const TodoInfo: React.FC<Result> = ({ title, completed, user }) => {
```

Reply:
Agree.